config/schemapatch/kueue-operator.crd.yaml

```diff
@@ -193,6 +225,60 @@ spec:
                                   additionalProperties:
                                     type: string
                               x-kubernetes-map-type: atomic
+                    manageJobsWithoutQueueName:
+                      description: |-
+                        ManageJobsWithoutQueueName controls whether or not Kueue reconciles
+                        jobs that don't set the annotation kueue.x-k8s.io/queue-name.
+                        If set to true, then those jobs will be suspended and never started unless
+                        they are assigned a queue and eventually admitted. This also applies to
+                        jobs created before starting the kueue controller.
+                        Defaults to false; therefore, those jobs are not managed and if they are created
+                        unsuspended, they will start immediately.
+                      type: boolean
+                    managedJobsNamespaceSelector:
+                      description: ManagedJobsNamespaceSelector can be used to omit some namespaces from ManagedJobsWithoutQueueName
```

Comment:
technically isn't it the inverse: 
``
ManagedJobsNamespaceSelector can be used to include some namespaces for ManagedJobsWithoutQueueName
```

Reply:
https://github.com/kubernetes-sigs/kueue/blob/main/apis/config/v1beta1/configuration_types.go#L42

No, this is correct.

managedJobsWithoutQueueName would add a label to all jobs to automatically add them into kueue.

And then ManagedJobsNamespaceSelector was added as that proved a bit too strong of a design.